examples/buildpacks/dual-storage-backend.md

```diff
@@ -80,7 +80,13 @@ tkn tr describe --last -o jsonpath="{.metadata.annotations.chains\.tekton\.dev/p
 tkn tr describe --last -o jsonpath="{.metadata.annotations.chains\.tekton\.dev/signature-taskrun-$TASKRUN_UID}" | base64 -d | jq
 ```
 
-Verify annotations with cosign:
+Verify the attestation with cosign 1.5.1+:
+
+```bash
+tkn tr describe --last -o jsonpath="{.metadata.annotations.chains\.tekton\.dev/signature-taskrun-$TASKRUN_UID}" | base64 -d > sig
+cosign verify-blob --key k8s://tekton-chains/signing-secrets --signature sig sig
+# Should output `Verified OK`
+```
 
 _In progress - see
```

Comment:
Are these still "in progress"? Isn't at least one of these closed?

Reply:
Oh good catch, I forgot this 🤦